### ISSUE TITLE
[ Bug ] Previews from other workflows are shown in the Sampler

### ISSUE THREAD

**Issue by JorgeR81** [chrisgoringe/cg-controller#464]
When we open a new workflow, the KSampler displays the "live preview" of the previous workflow. 

If we refresh the browser, it fixes the issue ( the previous "live preview" it's gone ).

**Comment by chrisgoringe** [comment]
You may notice that this sometimes happens with ComfyUI itself. The signals from the server to the UI don't clearly identify the workflow they are associated with...

**Comment by JorgeR81** [comment]
> You may notice that this sometimes happens with ComfyUI itself. The signals from the server to the UI don't clearly identify the workflow they are associated with...

 Yeah, I think I noticed this during the live preview, if I switch workflows.

But this is a different case:

- It happens in the completed live preview, after the generation.  
- It only happens in the Controller, not the node's graph. 
- It does not seem to happen in the `main` branch, only on `dev`

**Comment by chrisgoringe** [comment]
note to self: clear the image manager memory in new_workflow()

**Comment by chrisgoringe** [comment]
see if it works now

**Comment by JorgeR81** [comment]
No, I still have this issue.
